app/scripts/views/force_auth.js

```diff
@@ -136,8 +141,8 @@ define(function (require, exports, module) {
       };
     },
 
-    events: _.extend(SignInView.prototype.events, {
```

Comment:
This was an error earlier, the only reason we didn't come across it is because the signin page has no links to `/confirm_reset_password`. Unfortunately the signin page does have a link to `/reset_password` and this handler overwrote the signin variant for the signin screen.